### ISSUE TITLE
Form elements do not have associated label in ocr -> document_types -> submit

### ISSUE THREAD

**Issue by michaelasp** [CMU-313/Mayan-EDMS#60]
Error as follows

Failing Elements

`input<input class="select2-search__field" type="search" tabindex="0" autocomplete="off" autocorrect="off" autocapitalize="none" spellcheck="false" role="searchbox" aria-autocomplete="list" placeholder="" style="width: 0.75em;">`

Lighthouse score for accesibility is 76

**Comment by michaelasp** [comment]
Closed as duplicate
